ctru-rs/src/services/cam.rs

```diff
@@ -0,0 +1,1027 @@
+use bitflags::bitflags;
+use ctru_sys::Handle;
+
+pub struct Cam(());
+
+bitflags! {
+    #[derive(Default)]
+    pub struct CamPort: u32 {
+        const PORT_NONE = ctru_sys::PORT_NONE;
```

Comment:
Since the enum is named CamPort, you can remove the `PORT_` prefix from all variants.